### PR TITLE
Better dark email theme

### PR DESCRIPTION
**Pull request by hatch01**

### CODE REVIEW

File: apps/onyx/lib/screens/agenda/widgets/one_day_view.dart [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                  padding: const EdgeInsets.only(
+                    left: 20,
+                    right: 20,
+                    top: 15,
+                  ),
+                  child: Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      children: [
+                        Text(
+                          "coucou",
```
Comment (Persson-dev) [inline apps/onyx/lib/screens/agenda/widgets/one_day_view.dart:50]: Maybe this was not intended to stay there

**Comment by Persson-dev** [comment]
image
It does not build

File: apps/onyx/lib/screens/agenda/widgets/one_day_view.dart (hunk 1 above)
Comment (hatch01) [reply apps/onyx/lib/screens/agenda/widgets/one_day_view.dart:50]: It is also an error of commit in the wrong branch

**Comment by hatch01** [comment]
> image It does not build

it's because I did not published the mail library modification to pub.dev. So when you build as release your not using the last version.
run a `melos bs` to be able to build.
I will publish everithing and bump all the version for the publication before merging to main

**Comment by Persson-dev** [comment]
image
It does not properly load mails

**Comment by hatch01** [comment]
> image It does not properly load mails

fixed with 417e5ed3

**Pull request merged**